From 40be5ff4ecf3e40de2ed8d166bf7c18a78341fed Mon Sep 17 00:00:00 2001
From: shengfei Xu <xsf@rock-chips.com>
Date: Sat, 6 Oct 2018 19:19:43 +0800
Subject: [PATCH] arm64: dts: rockchip: rk1808-evb: change vcc_buck5 voltage

to fix up:
[    0.441016] vcc_buck5: Bringing 2800000uV into 2500000-2500000uV
[    0.442180] vcc_buck5: failed to apply 2500000-2500000uV constraint(-22)
[    0.442215] rk808 0-0020: failed to register 4 regulator
[    0.443429] rk808-regulator: probe of rk808-regulator failed with error -22

Change-Id: Ic36125718f7ace9ad498b45933f1a20a7fa4407d
Signed-off-by: shengfei Xu <xsf@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk1808-evb.dtsi | 8 +++++---
 1 file changed, 5 insertions(+), 3 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk1808-evb.dtsi b/arch/arm64/boot/dts/rockchip/rk1808-evb.dtsi
index e4b7b80a3f1a..bcee06e677bb 100644
--- a/arch/arm64/boot/dts/rockchip/rk1808-evb.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk1808-evb.dtsi
@@ -431,6 +431,8 @@
 			};
 
 			vccio_sd: LDO_REG8 {
+				regulator-always-on;
+				regulator-boot-on;
 				regulator-min-microvolt = <1800000>;
 				regulator-max-microvolt = <3300000>;
 
@@ -455,12 +457,12 @@
 			vcc_buck5: DCDC_REG5 {
 				regulator-always-on;
 				regulator-boot-on;
-				regulator-min-microvolt = <2500000>;
-				regulator-max-microvolt = <2500000>;
+				regulator-min-microvolt = <2200000>;
+				regulator-max-microvolt = <2200000>;
 				regulator-name = "vcc_buck5";
 				regulator-state-mem {
 					regulator-on-in-suspend;
-					regulator-suspend-microvolt = <2500000>;
+					regulator-suspend-microvolt = <2200000>;
 				};
 			};
 
-- 
2.35.3

